microservice/lpwan-custom-codec/src/test/java/com/cumulocity/microservice/lpwan/codec/rest/CodecControllerContractTest.java

```diff
@@ -0,0 +1,59 @@
+package com.cumulocity.microservice.lpwan.codec.rest;
+
+import com.cumulocity.microservice.lpwan.codec.sample.Application;
+import com.cumulocity.microservice.lpwan.codec.sample.SwaggerConfiguration;
+import com.cumulocity.microservice.lpwan.codec.sample.TestBeanConfiguration;
+import io.github.robwin.swagger.test.SwaggerAssertions;
+import io.swagger.models.Swagger;
+import io.swagger.parser.SwaggerParser;
+import io.swagger.util.Yaml;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.IOUtils;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.MalformedURLException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+
+@RunWith(SpringRunner.class)
+// TestBeanConfiguration is required, as the DecoderService and EncoderService beans are required to startup the application and are required by the CodecController.
```

Comment:
Can you just try one thing. Move mocked bean classes from TestBeanconfiguration to here and test if it works so that we can remove that file

Reply:
TestBeanConfiguration has been removed